charts/cluster-warmer/templates/hpa.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: autoscaling/v1
```

Comment:
FWIW if you bump to autoscaling/v2beta2, you could optionally scale on memory as well (or anything from the Metrics API I believe)

Reply:
Yeah I thought about that but ultimately I don't need CPU scaling as the warmer is solely controlled via the replicas counter (it won't autoscale at all).